JesusFilm/core
# feat: eng 700 searchbar suggestions

# Description

### Issue

Need to render some suggestions to educate users on how to apply filters

[Link to Linear Todo](https://linear.app/jesus-film-project/issue/ENG-700/searchbar-suggestions)

### Solution

Create a suggestions component for the dropdown
Change the logic for opening and closing dropdown so that it is more consistent switching between both

<img width="926" alt="image" src="https://github.com/user-attachments/assets/eb7e774c-9cc3-4da0-9ead-d10e7b4fd96c">
<img width="888" alt="image" src="https://github.com/user-attachments/assets/94905683-a18b-4838-b0bd-a0e060dbac43">


# How to test

Should still be able to use languages dropdown as before
Should also be able to navigate between suggestions dropdown and languages dropdown
Should always see two suggestions
When a suggestion is clicked, it should apply those refinements
